sync_image_ghcr.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+set -e
+# Simple script that pushes missing/outdated tags from an image in DockerHub to GHCR, skipping up-to-date tags 
+# GitHub package registry. You need to be logged in using 'docker login' first: https://docs.github.com/es/packages/working-with-a-github-packages-registry/working-with-the-container-registry
+target_repo="ghcr.io"
+tag_file="tag_list.txt"
+original_image="$1"
+rm -rf $tag_file
+url="https://hub.docker.com/v2/repositories/${original_image}/tags"
+tag_count=$(wget -q "$url" -O - | jq -r '.count')
```

Comment:
all the use-cases in this script scream "use `curl`" rather than "use `wget`", but that's just me :sweat_smile: (so this is personal preference)

Reply:
Oh yes I agree, in fact thought the same after finishing testing this 😝. Not sure why I went with wget in the first place, was just comfortable with it I guess, but it works™.

Reply:
both tools have their legitimate application use-cases, this one just happens to scream more `curl` than `wget`. 
This isn't a blocker though, as it really is just whatever you are more comfortable with in the end and that works :+1: